app/mailers/student_mailer.rb

```diff
@@ -21,7 +21,44 @@ def draw_invitation(user, college)
   # @param college [College] the college to pull settings from
   def selection_invite(user, college)
     @user = user
-    @college = college
+    @college = college || College.new
     mail(to: @user.email, subject: 'Time to select a suite!')
   end
+
+  def disband_notification(user, college)
+    @user = user
+    @college = college || College.new
+    mail(to: @user.email, subject: 'Your housing group has been disbanded')
+  end
+
+  def finalizing_notification(user, college)
+    @user = user
+    @college = college || College.new
+    @finalizing_path = if user.group.draw
+                         draw_groups_url(user.group)
+                       else
+                         groups_url(user.group)
+                       end
+    mail(to: @user.email, subject: 'Your housing group is now finalizing')
```

Comment:
I think we can change this to "Please confirm your housing group" --> I think we're getting rid of the 'finalizing' language anyway in #352